data/outport/common_test.go

```diff
@@ -55,7 +55,7 @@ func TestGetBody(t *testing.T) {
 	require.Nil(t, receivedBody)
 	require.Equal(t, errNilBodyHandler, err)
 
-	var body data.BodyHandler
+	var body data.BodyHandler // nolint
```

Comment:
do we need `// no lint` here and in transaction_test.go ?

Reply:
pushed the proper way